.github/workflows/scala.yml

```diff
@@ -8,14 +8,19 @@ on:
 
 jobs:
   build:
-
-    runs-on: ubuntu-latest
+    name: ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macOS-latest]
```

Comment:
Why do we need to run on macOS?

Reply:
because there's a pretty large amount of our broader developer base who use MacOS as their dev environment. It would be good to exercise that to verify that there are no platform issues breaking the build on that OS.

Reply:
Oh I see, thank you!